lighthouse/lighthouserc.js

```diff
@@ -0,0 +1,100 @@
+const AUDIT_URLS = {
+  FRONT_PAGE: {
+    live: ['https://test.bbc.com/arabic'],
+    test: ['https://www.test.bbc.com/arabic'],
+  },
+
+  IDX: {
+    live: [],
+    test: [
+      'https://www.test.bbc.com/persian/afghanistan',
+      'https://www.test.bbc.com/azeri/azerbaijan-23066612',
+    ],
+  },
+
+  LIVE_RADIO: {
+    live: ['https://www.bbc.com/korean/bbc_korean_radio/liveradio'],
+    test: ['https://www.test.bbc.com/korean/bbc_korean_radio/liveradio'],
+  },
+
+  OD_RADIO: {
+    live: [],
+    test: [
+      'https://www.test.bbc.com/pashto/bbc_pashto_radio/programmes/p056fcjb',
+    ],
+  },
+
+  OD_TV: {
+    live: [],
+    test: [
+      'https://www.test.bbc.com/pashto/bbc_pashto_tv/tv_programmes/w13xttn4',
+    ],
+  },
+
+  MAP: {
+    live: [],
```

Comment:
Do we need to find some URLs for these? Should we have a MAP for a popular service such as Mundo / Russian (perhaps with ads?)